scripts/generate-testcases.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
```

Comment:
I know it's missing on plenty of places but we should have a COPYLEFT header on the new files to start with that slowly.

Reply:
Updated